scripts/vim-patch.sh

```diff
@@ -312,8 +312,10 @@ submit_pr() {
   pr_message="$(printf '[RFC] vim-patch:%s\n\n%s\n' "${pr_title#,}" "${pr_body}")"
 
   if [[ $push_first -ne 0 ]]; then
-    echo "Pushing to 'origin/${checked_out_branch}'."
-    output="$(git push origin "${checked_out_branch}" 2>&1)" &&
+    local push_remote
+    push_remote="$(git config --get github.user || echo origin)"
+    echo "Pushing to '${push_remote}/${checked_out_branch}'."
+    output="$(git push --set-upstream "${push_remote}" "${checked_out_branch}" 2>&1)" &&
```

Comment:
Shouldn't upstream be `neovim/neovim`'s master branch?

Reply:
Hmm.. `git push --set-upstream foo bar"` pushes "bar" to "foo".
`--set-upstream` takes no args, but records "foo/bar" as the upstream.
Typically this allows for easier `git push` later on - where you do not want to push to neovim/neovim, but rather your PR's branch.

Reply:
Ah, I'm used to having `remote.pushdefault` set to my remote and doing `git checkout -b ... upstream/master` (which automatically sets the upstream branch) to create a branch based on neovim/neovim@master.  That way pulls come from neovim/neovim but pushes go to my repo by default.